src/components/views/voip/CallView.tsx

```diff
@@ -197,6 +199,11 @@ export const Lobby: FC<LobbyProps> = ({ room, connect, children }) => {
         }
     }, [connect, setConnecting]);
 
+    const tooltip = useMemo(() => {
+        if (connecting) return _t("Connecting");
+        if (callIsFull) return _t("Sorry - this call is currently full");
+    }, [connecting, callIsFull]);
```

Comment:
What's the rationale for `useMemo` here, and on line 342? These don't seem like expensive operations.

Reply:
Is there a reason not to use it?

Reply:
As the [memo all the things](https://attardi.org/why-we-memo-all-the-things/) argument goes, not really. And in the future React is supposed to move to automatic memoization of everything, anyways